docs/asciidoc/index.asciidoc

```diff
@@ -71,6 +71,13 @@ https://github.com/reactor/reactor-core/edit/master/docs/asciidoc/apdx-operatorC
 to "<<which-operator>>"
 endif::[]
 
+ifeval::["{backend}" == "html5"]
+[appendix]
```

Comment:
The include is guarded by the `ifeval`, is that on purpose?

Reply:
(it was on purpose to completely skip that appendix in the PDF when I couldn't find a way to have all graphics render, but now that issue has been fixed)